config/presets/payer_server_run_against_smart_app_tests.json

```diff
@@ -24,10 +24,13 @@
     },
     {
-      "name": "credentials",
+      "name": "smart_auth_info",
       "title": "OAuth Credentials",
-      "type": "oauth_credentials",
-      "value": "{\"access_token\":\"eyJhbGciOiJub25lIn0.eyJpbmZlcm5vX2NsaWVudF9pZCI6InNhbXBsZSJ9\"}"
+      "type": "auth_info",
+      "value": {
+        "auth_type": "public",
```

Comment:
I'm not sure what auth type this should actually be.

Reply:
This would be `backend_services`, right? The way it's implemented it just needs a bearer token

Reply:
Yeah, I didn't think it would matter from a technical perspective, but it would be nice for the auth type to make sense for the user.